### ISSUE TITLE
Should it be (image_rows, image_cols)?

### ISSUE THREAD

**Issue by zihaozhihao** [jocicmarko/ultrasound-nerve-segmentation#81]
https://github.com/jocicmarko/ultrasound-nerve-segmentation/blob/df9eb77f0ac3b84ae125b6b177ef2e4429818d66/submission.py#L11

According to the scikit-image resize api, the output_shape should first be rows, then cols.

**Comment by GeorgeBatch** [comment]
Yes. I think it occurred because, in cv2.resize(), which was used in an earlier version, the arguments were switched. https://docs.opencv.org/2.4/modules/imgproc/doc/geometric_transformations.html#void%20resize(InputArray%20src,%20OutputArray%20dst,%20Size%20dsize,%20double%20fx,%20double%20fy,%20int%20interpolation)
